scripts/create_config.js

```diff
@@ -25,7 +25,7 @@ const NCT_ETH_EXCHANGE_RATE = 80972;
 
 // See docker setup
 const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000';
-logger.info(`Logging format: ${args.log_format || 'text'}`);
+logger.info(`Logging format: ${args.log_format == 'json' ? 'json' : 'text'}`);
```

Comment:
Let's not duplicate logic here. If we add additional logging formats beyond `json` and `test`, we'd need to make mods here and here: https://github.com/polyswarm/contracts/blob/72b880fc7cdf51f52968e01c81783be4eadd5def/scripts/logger.js#L19

Let's canonicalize the log format in a single place and pass the canonicalized log format to `logger.js`

Reply:
Put another way, this would easily mask a bug. If we introduced `foobar` as a logging format, `foobar` would be passed to `logger.js`, but this would continue to say `Logging format: text`. Let's avoid patterns where updates need to happen in multiple places.

Reply:
Agreed, I've moved all of this logic into `logger.js` itself and stored it on the `logger` object for public use.